exonum-java-binding/core/rust/src/proxy/node.rs

```diff
@@ -66,7 +67,10 @@ impl Node {
         let verified = Verified::from_value(tx, pub_key.to_owned(), secret_key);
         let tx_hash = verified.object_hash();
         // TODO(ECR-3679): check Core behaviour/any errors on service inactivity
-        self.blockchain.sender().broadcast_transaction(verified)?;
+        self.blockchain
+            .sender()
+            .broadcast_transaction(verified)
+            .wait()?;
```

Comment:
`broadcast_transaction` now returns a Future! However, the actual implementation is unchanged - it used to call `wait` in the Core code, now we call it ourselves.